pyproject.toml

```diff
@@ -1,14 +1,15 @@
 [tool.poetry]
 name = "GQLAlchemy"
-version = "1.4.0"
+version = "1.5.0"
```

Comment:
this is going to 1.4.1

Reply:
Done

Reply:
In the end, just make sure this goes to 1.4.1, like @Josipmrden said :)